draft-regext-rdap-extensions.md

```diff
@@ -724,24 +807,24 @@ This document does not change the RDAP JSON Values Registry nor its purpose.
 However, this document does update the procedures for registrations and the
 processes to be used by its expert reviewers.
 
-In addition to the registration of values, RDAP extensions defined by the IETF
-and other IETF specifications MAY define additional value types (the "type" field), 
-however these specifications MUST describe the specific JSON field to be used 
-for each new value type. 
+In addition to the registration of values, RDAP extensions defined by
+the IETF and other IETF specifications MAY define additional value
+types (the "type" field).  These specifications MUST describe the
+specific JSON field to be used for each new value type.
 
 [@!RFC9083, Section 10.2] defines the criteria for the values. Of these, criteria two
```

Comment:
There were a few of these using capitals, and each has now been changed to lowercase, so that the text becomes "Section ... of RFC ...", rather than just "Section ...".